particula/aerosol_dynamics/parcel.py

```diff
@@ -98,11 +113,15 @@ def particle_radius(self) -> float:
         """Returns list of radi of particles"""
         return [i.radius() for i in self.particle_classes()]
 
+    def particle_density(self) -> float:
+        """Returns list of densities of particles."""
+        return [i.density() for i in self.particle_classes()]
+
     def particle_charge(self) -> float:
         """Returns list of charges of particles."""
         return [i.charge() for i in self.particle_classes()]
 
-    def knudsen_number_particle(self) -> float:
+    def particle_knudsen_number(self) -> float:
```

Comment:
Need a different. Please think of a naming scheme that can be consistent throughout?

Reply:
What is wrong, I based it on the particle class methods?

Reply:
The name is confusing / too similar to the name used in the particle class, that's what I meant. So maybe we should distinguish it better? `knudsen_number_list` maybe? Or something like that

Reply:
I like particle as the prefix, so when we add gases it would be gases_vapor_pressures_list. So sufix of _list and making number and radius plural, should be clearer

Reply:
Alright, thanks!